project1-sps/script.js

```diff
@@ -1,4 +1,92 @@
+// initial mode & username variable
+var currentMode = "Enter Name";
+var userName = ""; // empty string, nothing stored
+
+// record user's Win Loss Draw score
+var userWinRecord = 0;
+var userLoseRecord = 0;
+var userDrawRecord = 0;
+
+// computerOption helper function
+var computerSPS = function (input) {
+  var randomNumber = Math.floor(Math.random() * 3) + 1;
+  console.log(randomNumber);
+  if (randomNumber == 1) {
+    return "scissors";
+  } else if (randomNumber == 2) {
+    return "paper";
+  } else {
+    return "stone";
+  }
+};
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+  var myOutputValue;
```

Comment:
Since we are not using `myOutputValue` in the first game mode, we can define it inside the code block of the second game mode (so that we don't define the variable _"for nothing"_)